src/components/Hero.jsx

```diff
@@ -0,0 +1,161 @@
+import { PersonalizedText } from './shared/typography';
+import { Paragraph } from './shared/typography';
+
+const Hero = () => {
+	return (
+		<>
+			<div className='flex flex-col md:flex-row'>
+				<div className='w-[100%] md:w-1/2 '>
+					<div className='h-1/3 flex flex-col justify-end ml-2' style={{ color: '#FF4409' }}>
+						<PersonalizedText>Welcome family...</PersonalizedText>
+					</div>
+					<div className='h-1/3'>
+						<Paragraph style={{ fontSize: '75px', fontWeight: '500', letterSpacing: '0.75px', lineHeight: 'normal' }}>
+							Lorem Ipsum Dolar sit Amet Luyto
+						</Paragraph>
+					</div>
+					<div className='h-1/4'>
+						<button
+							className='px-4 py-2 rounded-md border border-solid border-black bg-orange-500 shadow-button mt-20 md:mt-35'
+							style={{
+								boxShadow: '2px 2px 0px 0px #000, 3px 4px 9.2px 0px rgba(222, 222, 222, 0.48) inset',
+								fontFamily: 'Syne, sans-serif',

```

Comment:
Please use typography here instead of defining styles.

Reply:
changed https://github.com/dscnitrourkela/project-udon/pull/17/commits/537e17d76660156681afe3dabb9cec6f44c6c672